src/dom/dom.js

```diff
@@ -359,16 +516,42 @@ p5.prototype.createP = function (html = '') {
 };
 
 /**
- * Creates a `&lt;span&gt;&lt;/span&gt;` element in the DOM with given inner HTML.
+ * Creates a `&lt;span&gt;&lt;/span&gt;` element. It's commonly used as a
+ * container for other elements.
```

Comment:
We also say this for `<div>`, maybe we can add something small about it being generally used as a container for inline things? Not sure what the best wording for that is. Things that might wrap along with text?